dask/array/reductions.py

```diff
@@ -408,8 +408,7 @@ def moment_chunk(A, order=2, sum=chunk.sum, numel=numel, dtype='f8', **kwargs):
     total = sum(A, dtype=dtype, **kwargs)
     n = numel(A, **kwargs).astype(np.int64)
     u = total / n
-    diff = A - u
-    xs = [((A - u)**i).sum(dtype=dtype, **kwargs) for i in range(2, order + 1)]
+    xs = [sum((A - u)**i, dtype=dtype, **kwargs) for i in range(2, order + 1)]
```

Comment:
`sum(...)` is much slower than `(...).sum()` for cases where the former works.

Reply:
Unless you're using `np.sum`...

Reply:
This code is misleading, this should probably change at some point. Check my comment: https://github.com/dask/dask/pull/4525#discussion_r264689869.